main.js

```diff
@@ -178,4 +178,213 @@ const data = [
         hotel: 'Hotel Rehberge Berlin Mitte',
     },
 ];
-console.log(searchString('Mitte', data));
\ No newline at end of file
+console.log(searchString('M', data));
+
+function checkPalindrome(str) {
+return str.split('').reverse().join('') === str;
+}
+console.log(checkPalindrome('poloolop'))
+
+function searchHotels(str, hotels) {
+    let result = hotels.filter(function (e) {
+        return e.country.includes(str) || e.city.includes(str) || e.name.includes(str);
+    }).map(function (e){
```

Comment:
you could do .map(e => e.country + ', ' + e.city +', ' + e.name);
this looks simpler, don't you think so?